### PR TITLE
Dev/last fixes

### PR DESCRIPTION
**Pull request by AdrienRem**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/contamination/fragment/UserInfectionFragment.java [hunk 1]
```diff
@@ -235,25 +229,15 @@ private void executeHealthStatusChange() {
     }
 
     private void sendRecoveryToFirebase() {
-        // TODO: @Lucie I think that doesn't upload anything to Firestore
+        // TODO: I think that doesn't upload anything to Firestore
+        // --->> Are you sure? From what I read on the doc,
+        //                      update method from DocumentReference updates on Firebase
```
Comment (AdrienRem) [inline app/src/main/java/ch/epfl/sdp/contamination/fragment/UserInfectionFragment.java:234]: From my tests of the app the infection status seems to be preserved accross app launchs

**Review by AdrienRem — COMMENTED** [review]
All good ! Thank you for removing all the hardcoded strings

File: app/src/main/java/ch/epfl/sdp/contamination/fragment/UserInfectionFragment.java [hunk 2]
```diff
@@ -235,25 +229,15 @@ private void executeHealthStatusChange() {
...
         // TODO: [LOG]
         Log.e("RECOVERY_SENDER", account.getId());
         DocumentReference ref = documentReference(privateUserFolder, account.getId());
         ref.update(privateRecoveryCounter, FieldValue.increment(1));
     }
 
-    private CompletableFuture<Boolean> retrieveUserInfectionStatus() {
```
Comment (AdrienRem) [inline app/src/main/java/ch/epfl/sdp/contamination/fragment/UserInfectionFragment.java:245]: The user status is stored locally now that's correct ?

Comment (matteopariset) [reply app/src/main/java/ch/epfl/sdp/contamination/fragment/UserInfectionFragment.java:245]: Also. Both locally and remotely

**Review by matteopariset — APPROVED** [review]

File: app/src/main/java/ch/epfl/sdp/contamination/fragment/UserInfectionFragment.java (hunk 2 above)
Comment (matteopariset) [reply app/src/main/java/ch/epfl/sdp/contamination/fragment/UserInfectionFragment.java:245]: But why do we need this method?
The status should be taken from the Carrier initialized by LocationService

**Review by matteopariset — CHANGES_REQUESTED** [review]
Remove retrieveUserInfectionStatus

**Review by lucieho — APPROVED** [review]

**Comment by lucieho** [comment]
> Remove retrieveUserInfectionStatus

This method was used when we used to retrieve the user's status from firebase each time we would open this fragment
It is not used anymore
From what I understood, the state is saved differently now
Unless you were thinking of using it later on, it should be fine

**Review by matteopariset — APPROVED** [review]

**Pull request merged**